packages/dappmanager/src/modules/ethClient/changeEthMultiClient.ts

```diff
@@ -43,3 +61,80 @@ export async function changeEthMultiClient(
     eventBus.runEthClientInstaller.emit();
   }
 }
+
+// Utils
+
+export async function migrateEthClientFullNode(
+  dnpName: string,
+  remove: boolean
```

Comment:
Not a big fan of using booleans to completely change the behaviour of the function. Try to split it into two functions: adding and removing and de-duplicating logic between them